src/modules/localization/src/wheel_odom/odometry.cpp

```diff
@@ -3,9 +3,7 @@
 /*
 Takes wheel odometry from Arduino encoders and translates to vehicle velocity and position
 NOTE: This script is to handle the raw wheel odometry values from the Signwise 600 P/R rotary encoder.
-      It also takes in various inputs from topics yaw from the IMU to dead-reckon position and velocity in the X and Y frame.
-      Dead-reckoning requires previous pose to be tracked, we take advantage of this by subscribing to
-      the output of the EKF and using those pose messages as the previous pose of the wheel odometry.
+      It also takes in various inputs from topics yaw from the IMU to calculate velocity in the X and Y frame.

```

Comment:
It looks like IMU subscription has been commented out. Is this documentation still valid?

Reply:
I'll edit the documentation and delete the excess comments, the transform will just be done within the `robot_localization` package.